gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 # Note: Check https://gradle.org/release-checksums/ before updating wrapper or distribution
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=027fdd265d277bae65a0d349b6b8da02135b0b8e14ba891e26281fa877fe37a2
+#distributionSha256Sum=027fdd265d277bae65a0d349b6b8da02135b0b8e14ba891e26281fa877fe37a2
```

Comment:
Why not just delete this?  Actually do you know why it's there in the first place?  I don't see it referenced anywhere else

Reply:
I believe it's a mechanism to verify that we're building with the right gradle and not something that's MITMing our build.

Reply:
Right, but where is it used?  Why did you comment it out and why not just delete it if you're going to comment it out?

Reply:
I've put in a comment